Client/Frontend/UserContent/UserScripts/YoutubeAdblock.js

```diff
@@ -34,3 +35,119 @@
         },
     });
 })();
+
+/// SetJS
+(function() {
+    const chain = '{{1}}';
+    let cValue = '{{2}}';
```

Comment:
@Brandon-T this is _almost_ right :smile: 

the first instance each of `{{1}}`, `{{2}}`, etc. in a scriptlet is a template parameter, it should get filled in with whatever arguments are used in the `+js(...)` rule (if any).

In this case since you have the two following rules:
```
youtube.com,youtube-nocookie.com##+js(set, ytInitialPlayerResponse.adPlacements, undefined)
youtube.com,youtube-nocookie.com##+js(set, playerResponse.adPlacements, undefined)
```

so you'd need this whole `SetJS` construct _twice_. The first one will have:
```
const chain = 'ytInitialPlayerResponse.adPlacements`;
let cValue = 'undefined';
```

and the second will have:
```
const chain = 'playerResponse.adPlacements';
let cValue ='undefined';
```

For future reference, you only ever change the first occurrence of a template parameter; the same numbered parameter might appear multiple times in the scriptlet but the 2nd+ ones are meant to be interpreted as literal strings to check if the argument is set. Also, it's totally fine if there is e.g. a `{{3}}` in the scriptlet but only 2 arguments provided in `+js(...)` - you'd just not replace the `{{3}}`.

Reply:
Fixed. I made it parameterized so I don't have to define the function twice. I'd just call it twice with different parameters.